packages/jest-config/src/normalize.ts

```diff
@@ -982,10 +982,10 @@ export default function normalize(
     if (newOptions.collectCoverageFrom) {
       collectCoverageFrom = collectCoverageFrom.reduce((patterns, filename) => {
         if (
-          !micromatchSome(
-            replacePathSepForGlob(path.relative(options.rootDir, filename)),
+          !micromatch(
+            [replacePathSepForGlob(path.relative(options.rootDir, filename))],
             newOptions.collectCoverageFrom!,
-          )
+          ).length
```

Comment:
Could we do `>0` instead of `!length`?

Reply:
Since it's a negation, I went with `.length === 0`